frontend/src/components/layout/Tabbing.vue

```diff
@@ -50,6 +59,7 @@ export default {
       currentTab: this.initialTab
     }
   },
+  expose: ['currentTab'],
```

Comment:
Might be true, since I altered the behavior. I just wasn't 100% where else this might be used, so I left it anyways. But you're right, it's probably not needed.